db-client-java/src/main/java/com/eventstore/dbclient/ServerVersion.java

```diff
@@ -4,6 +4,9 @@ class ServerVersion {
     private final int minor;
     private final int patch;
 
+    // old server with no support for server features api
+    public static ServerVersion Old = new ServerVersion(0, 0, 0);
```

Comment:
I would rather keep the current logic where if there is no server version available, we don't return one with bogus values in it.

Reply:
@YoEight ok, does this imply changing the signature to return `Optional<ServerVersion>` instead of `ServerVersion`?
```
public Optional<ServerVersion> getServerVersion()
```

Reply:
I would prefer yes.

Reply:
fixed.